autopush/utils.py

```diff
@@ -410,15 +415,10 @@ def from_message_table(cls, uaid, item):
 
     @classmethod
     def from_webpush_request_schema(cls, data, fernet):
+        # type: (WebPushRequestSchema, Fernet) -> WebPushNotification
```

Comment:
data is actually just a dict not a schema. then there's no circular import problem